### PR TITLE
Spellcheck and mdlint

### PR DESCRIPTION
**Pull request by pdehaan**
## Description (optional)

- Noticed a few typos and incomplete sentence.

### CODE REVIEW

File: docs/android-frontend-testing.md [hunk 1]
```diff
@@ -97,13 +97,13 @@ These fields affect which OS, app and build the experiment is for. These should
 
 These fields affect segment the population for eligibility for the experiment, and which branch they'll be given.
 
- * `targeting`: Experimenter will help you generate this JEXL query string. If Nimbus evaluates this on a given device to `true`, then the device is eligible for the experiment. If your testing the app, then you 
+ * `targeting`: Experimenter will help you generate this JEXL query string. If Nimbus evaluates this on a given device to `true`, then the device is eligible for the experiment. If your testing the app, then you (TODO)
```
Comment (pdehaan) [inline docs/android-frontend-testing.md:100]: Notice this looks like a partial thought. Once/if this lands I can file a new issue in mozilla/experimenter to add more content.

File: docs/android-frontend-testing.md [hunk 2]
```diff
@@ -142,4 +142,4 @@ You can then vary the device (e.g. the locale or language), or by using a small
 
 ## Conclusion
 
-Much of these will be ameliorated by testing tooling. We appreciate that the current situation is less than ideal.
\ No newline at end of file
+Much of these will be ameliorated by testing tooling. We appreciate that the current situation is less than ideal.
```
Comment (jodyheavener) [inline docs/android-frontend-testing.md:145]: Can't see what the difference is here, even after a good squint 🤨

**Review by jodyheavener — APPROVED** [review]
🤠

File: docs/android-frontend-testing.md (hunk 2 above)
Comment (pdehaan) [reply docs/android-frontend-testing.md:145]: Sorry. I think this was just kvetching that there was no newline at the end of the file. 🤷 
I should probably just disable that rule in my local mdlint config file.

**Pull request merged**